src/app/public/modules/action-hub/action-hub.component.html

```diff
@@ -0,0 +1,34 @@
+<sky-wait
+  [isFullPage]="true"
+  [isWaiting]="loading"
+></sky-wait>
+<sky-fluid-grid *ngIf="!loading">
+  <sky-row>
+    <sky-column>
+      <h1 class="sky-font-heading-1">{{ title }}</h1>
+    </sky-column>
+  </sky-row>
+  <sky-row>
+    <sky-column screenXSmall="12" screenLarge="9">
+      <div class="sky-margin-stacked-lg">
+        <ng-content select="sky-action-hub-buttons"></ng-content>
+      </div>
+
+      <div class="sky-margin-stacked-xxl">
+        <sky-needs-attention [items]="needsAttention"></sky-needs-attention>
+      </div>
+
+      <ng-content select="sky-action-hub-content"></ng-content>
+    </sky-column>
+    <sky-column screenXSmall="12" screenLarge="3">
+      <sky-link-list
+        title="Related Links"
```

Comment:
This and the `Recently Accessed` should use localization with the `skyLibResources` pipe.  Let me know if you haven't done any of that yet and need me to walk you through it.